.github/workflow/ci.yaml

```diff
@@ -0,0 +1,35 @@
+
+name: Compilation Workflow
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    env:
+      GO111MODULE: auto
+      TARGET: amd64
+        
+    strategy:
+      matrix:
+        go-version: [1.13.x]
+        os: [ubuntu-18.04]
```

Comment:
Should we include 16.04 too till the time we support it?

Reply:
added it to the list